Rakefile

```diff
@@ -1,8 +1,6 @@
 require "bundler/gem_tasks"
 require "rspec/core/rake_task"
 
-RSpec::Core::RakeTask.new(:spec) do |task|
-  task.ruby_opts = "-W"
```

Comment:
I only removed it because of the **incredibly stupid omg rubby why you so dumb** warning about private `attr_reader`s (one of which I use in `SafeWebhook`.

Is there a better way to handle that philosophical disagreement I have with rubby?

I get that this codebase had some cruft that was found via `-W`, but we don't run with warnings on typically, and now that this codebase was  brought inline, maybe it's no longer needed?